pyfr/solvers/euler/kernels/flux.mako

```diff
@@ -25,3 +25,25 @@
     f[${i}][${j + 1}] = rhov[${i}]*v[${j}]${' + p' if i == j else ''};
 % endfor
 </%pyfr:macro>
+
+<%pyfr:macro name='inviscid_1dflux' params='s, f, p, v'>
```

Comment:
Anything we can do to eliminate this kernel, or at least reduce the amount of code duplication?  Open to ideas.